gradle/dependencies.gradle

```diff
@@ -19,14 +19,14 @@ ext.versions = [
 
 ext.deps = [
     plugins: [
-        android: 'com.android.tools.build:gradle:3.4.1',
+        android: 'com.android.tools.build:gradle:3.6.0',
```

Comment:
This is not yet a `compileOnly` dependency so we'd be forcing this on our consumers. Are we okay with that?

Reply:
Could we have different versions for build classpath and dependency?

Reply:
Sure. That's an easy workaround.